libs/auth/src/cac/common_access_card.ts

```diff
@@ -108,12 +117,18 @@ export class CommonAccessCard implements CommonAccessCardCompatibleCard {
   private readonly cardReader: CardReader;
   private cardStatus: CardStatus<CommonAccessCardDetails>;
   private readonly customChallengeGenerator?: () => string;
+  private readonly certPath?: string;
```

Comment:
This is only here for `createAndStoreCert`. Remove this and make it a parameter to that method?

Reply:
I see your point but... this approach mirrors the approach taken on `JavaCard`, so I think best to leave it for consistency?

Reply:
Sure, that's fine.